client/src/app/components/stardust/block/BlockTangleState.tsx

```diff
@@ -13,7 +13,8 @@ const BlockTangleState: React.FC<BlockTangleStateProps> = ({ network, status, mi
 
     useEffect(() => {
         if (milestoneDetails?.milestone?.timestamp) {
-            setAgo(moment(milestoneDetails.milestone?.timestamp * 1000).fromNow());
+            const readableTimestamp = DateHelper.format(DateHelper.milliseconds(milestoneDetails.milestone.timestamp));
+            setAgo(readableTimestamp);
```

Comment:
Should we rename this react state now that it's not holding the "ago" string ?

Reply:
Sure thing!